model/testutil/stats.go

```diff
@@ -0,0 +1,257 @@
+package testutil
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/apimodels"
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/model/testresult"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type TaskStatus struct {
+	Status         string
+	DetailsType    string
+	DetailsTimeout bool
+	TimeTaken      time.Duration
+}
+
+var Success100 = TaskStatus{"success", "test", false, 100 * 1000 * 1000 * 1000}
+var Success200 = TaskStatus{"success", "test", false, 200 * 1000 * 1000 * 1000}
+var TestFailed = TaskStatus{"failed", "test", false, 20}
+var Timeout = TaskStatus{"failed", "test", true, 100}
+var SetupFailed = TaskStatus{"failed", "setup", false, 10}
+var SystemFailed = TaskStatus{"failed", "system", false, 10}
+
+func ClearCollection(s suite.Suite, name string) {
+	err := db.Clear(name)
+	s.Require().NoError(err)
+}
+
+func InsertTask(s suite.Suite, project string, requester string, taskId string, execution int,
```

Comment:
I'm not sure what, if any benefit this has over just constructing the documents where you needing and calling insert there? The signature for these functions is super difficult to use (and easier to confuse than just using the default constructors. 

(also in general, I'd prefer to return errors from functions rather than passing testify/testing objects around to handle errors.)

Reply:
changed.